packages/network-of-terms-catalog/catalog/datasets/gtaa_onderwerpen_beng.jsonld

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.beeldengeluid.nl/gtaa/OnderwerpenBenG",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "GTAA Onderwerpen beeld-geluid"

```

Comment:
This is a slightly weird name.

Reply:
That is what it is referred to. And indeed confusing. It has nothing to do with the Instituut voor Beeld en Geluid, but its just that terms from these lists are more focussing on things that can be seen (beeld) or can be heard (sound). The ohter Onderwerpen axis is abstracting away from physical and concrete stuff. E.g. When you see a 'Suzuki' the more general subject will be 'Cars'. The beeld-geluid axis would sooner accept more specific instances of concepts. I hope this example helps.

Reply:
It does. The name still needs the colon `:` for consistency.

Reply:
It has a colon locally. Can you check again? Maybe it was fixed in the latest push. We should be in sync.